tests/integration_tests/functional/test_net_config_space.py

```diff
@@ -174,15 +176,17 @@ def _send_data_g2h(ssh_connection, host_ip, host_port, iterations, data, retries
         "s.close()"
     )
 
+    # The guest has Python2
     cmd = 'python -c "{}"'.format(
```

Comment:
Would it be clearer to specify `python2` here?
```suggestion
    cmd = 'python2 -c "{}"'.format(
```

Reply:
hmm `python2` does exist in the rootfs, but I would rather leave it the way it is. We will have to change it in the future when we move to Ubuntu 22 for rootfs.